### PR TITLE
Add main files for html

### PR DESCRIPTION
**Pull request by NikitaBondarevv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    <div class="creator">
+      <img src="images/shapeFooter.png" alt="Shape">
+      <span>Copyright 2014 Maguwohost by FikriStudio</span>
+    </div>
+  </footer>
+</body>
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+<script src="pluginAnimation.js"></script>
+<script src="main.js"></script>
+</html>
```
Comment (iliatcymbal) [inline index.html]: add new line

File: pluginAnimation.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    const toggle = "toggle";
+    
+    this.animate({
+      height: toggle,
+      opacity: toggle
+    }, 1000);
+
+    return this;
+  };  
+})(jQuery);
```
Comment (iliatcymbal) [inline pluginAnimation.js]: add new line

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,1003 @@
...
+
+  .way-to-price a {
+    font-size: 12px;
+    padding: 15px 30px;
+  }
+
+  #footer {
+    padding: 79px 10px 160px;
+  }
+}
```
Comment (iliatcymbal) [inline styles.css]: add new line

**Review by iliatcymbal — CHANGES_REQUESTED** [review]
Please see comment

**Comment by NikitaBondarevv** [comment]
@iliatcymbal please check

**Comment by iliatsymbal** [comment]
Looks good to me

**Review by iliatsymbal — APPROVED** [review]

**Review by iliatcymbal — APPROVED** [review]

**Pull request merged**